### PR TITLE
Update paths for RefApp 2.x and platform files

### PR DESCRIPTION
**Pull request by kdaud**
[Update paths for RefApp 2.x and Platform files](https://issues.openmrs.org/browse/RATEST-320)

### CODE REVIEW

**Comment by sherrif10** [comment]
Did you forget to add a ticket for this change.

**Comment by kdaud** [comment]
I do not think the change requires to log a ticket. It's just about updating paths for RefApp 2.x and Platform files.

**Comment by sherrif10** [comment]
I think we can add it, Tickets are very important to track issues FYI.

**Comment by sherrif10** [comment]
@kdaud  Is this ticket updating the path or refactoring the path.

**Comment by kdaud** [comment]
Updating paths for RefApp 2.x and Platform files.

**Review by sherrif10 — COMMENTED** [review]
What is the difference between update and refactor. Can we say we have refactored version  1 to 2.

**Comment by kdaud** [comment]
`Refactoring` is introduction of modifications that over the time will improve the overall quality of the code base making it easier to maintain and evolve over the time.

`Software updates` is about modifying, fixing and altering the current software program. They’re the software equivalent of keeping a bicycle oiled,.It’s still the same bike, it just runs smoother.

**Comment by sherrif10** [comment]
> modifying, fixing and altering the current software program

**Comment by sherrif10** [comment]
oops appologies.

File: .github/workflows/platform-install-advanced.yml [hunk 1]
```diff
@@ -8,6 +8,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: qaframework-bdd-tests
```
Comment (sherrif10) [inline .github/workflows/platform-install-advanced.yml:13]: @kdaud  i see you are checking out the working directory which was the same thing done on line 46 , do you mind giving me the reason why.

Comment (kdaud) [reply .github/workflows/platform-install-advanced.yml:13]: It is smarter to set the working directory for all the commands to be executed than the current setting how the directory is checked out.

**Comment by kdaud** [comment]
@sherrif10 any more recommendation?

**Review by sherrif10 — APPROVED** [review]
Thanks @kdaud .

**Comment by sherrif10** [comment]
@kdaud can you take some time debugging platform tests and the corresponding tests as well, i see they are indicating red light.

**Pull request merged**